.clangd

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+CompileFlags:
+  Add: -Wno-unknown-warning-option
+  Remove: [--hack*, --ibt,--orc,--retpoline,--rethunk,--sls,--static-call,--uaccess,--link,--module,-mpreferred-stack*,-mindirect-branch*,-fno-allow-store*,-fconserve-stack,-mrecord-mcount,-ftrivial-auto*,-fmin-function-alignment=16]
```

Comment:
Are you sure this is really getting picked up? Same for the kernel clang-tidy config. See https://github.com/DaanDeMeyer/mkosi-kernel/blob/c589c2baa0db680ec52278f230246fafc75771be/mkosi.profiles/kernel/mkosi.clangd#L32-L43

Reply:
Yeah it's picked up, we run clangd in /work/src/kernel so it picks up .clang-format from /work/src/kernel and .clangd from /work/src this way. I didn't check .clang-tidy

Reply:
Ops, yeah clang-format is what I meant